### DISCUSSION TITLE
create new algorithms

### DISCUSSION THREAD

**Discussion by Asbaharoon** [community/community#28879] [Other Feature Feedback, Questions, & Ideas]
Good afternoon my friends 

I create files contains new my Job for Algorithms in my profile page and i hope you to check it and give me your opinion 

Thank you and Best Regards,

**Comment by Asbaharoon** [comment]
[point of sale system management.docx](https://github.com/community/community/files/9307379/point.of.sale.system.management.docx)
[Algorithm for Bank database project.docx](https://github.com/community/community/files/9307381/Algorithm.for.Bank.database.project.docx)
[new hospital system management.docx](https://github.com/community/community/files/9307383/new.hospital.system.management.docx)
i attached my 3 files for watching

**Reply by mariadhari6** [reply]
> Good afternoon my friends
> 
> I create files contains new my Job for Algorithms in my profile page and i hope you to check it and give me your opinion
> 
> Thank you and Best Regards,



> Good afternoon my friends
> 
> I create files contains new my Job for Algorithms in my profile page and i hope you to check it and give me your opinion
> 
> Thank you and Best Regards,

In my opinion, image files are better placed in the images folder, and the content files use markdown so that they can be read directly in the repository.

**Reply by Asbaharoon** [reply]
> [point of sale system management.docx](https://github.com/community/community/files/9307379/point.of.sale.system.management.docx) [Algorithm for Bank database project.docx](https://github.com/community/community/files/9307381/Algorithm.for.Bank.database.project.docx) [new hospital system management.docx](https://github.com/community/community/files/9307383/new.hospital.system.management.docx)                                       i hope any one to check my content and add remarks

**Comment by Asbaharoon** [comment]
i hope any one check my content and add remarks

**Comment by Asbaharoon** [comment]
[Algorithm for Bank database project.docx](https://github.com/community/community/files/9348372/Algorithm.for.Bank.database.project.docx)
[new hospital system management.docx](https://github.com/community/community/files/9348378/new.hospital.system.management.docx)
[point of sale system management.docx](https://github.com/community/community/files/9348379/point.of.sale.system.management.docx)
i hope any one to check my content and add remarks and Give me your impressions
